### PR TITLE
Fix/remove simularium engine specific code

### PR DESCRIPTION
**Pull request by ascibisz**
Time Estimate or Size
=======
small

Problem
=======
As we fully transition to octopus, it's time to strip out the simularium-engine specific code that is no longer useful
[Link to story or ticket](https://github.com/simularium/simularium-viewer/issues/389)

Solution
========
* got rid of the `--octopus` commandline option, as that will now be the assumed platform
* got rid of the `useOctopus` option, as that will now be the assumed platform

A website PR will need to be made after this, updating their `NetConnectionParams` to remove `useOctopus`

### CODE REVIEW

File: src/simularium/RemoteSimulator.ts [hunk 1]
```diff
@@ -12,15 +12,8 @@ import { ISimulator } from "./ISimulator";
...
-const enum PlayBackType {
-    ID_LIVE_SIMULATION = 0,
-    ID_PRE_RUN_SIMULATION = 1,
-    ID_TRAJECTORY_FILE_PLAYBACK = 2,
-    // insert new values here before LENGTH
-    LENGTH,
-}
-// a RemoteSimulator is a ISimulator that connects to the Simularium Engine
-// back end server and plays back a trajectory specified in the NetConnectionParams
+// a RemoteSimulator is a ISimulator that connects to the Octopus back end server
```
Comment (ShrimpCryptid) [inline src/simularium/RemoteSimulator.ts:15]:
```suggestion
// a RemoteSimulator is a ISimulator that connects to the Octopus backend server
```
nit

File: src/simularium/RemoteSimulator.ts [hunk 2]
```diff
@@ -12,15 +12,8 @@ import { ISimulator } from "./ISimulator";
 import { TrajectoryFileInfoV2, VisDataMessage } from "./types";
 import { TrajectoryType } from "../constants";
 
-const enum PlayBackType {
-    ID_LIVE_SIMULATION = 0,
-    ID_PRE_RUN_SIMULATION = 1,
-    ID_TRAJECTORY_FILE_PLAYBACK = 2,
-    // insert new values here before LENGTH
-    LENGTH,
-}
```
Comment (ShrimpCryptid) [inline src/simularium/RemoteSimulator.ts:21]: Why is this enum no longer needed? Is it only used by Simularium engine?

**Review by ShrimpCryptid — APPROVED** [review]
Had a few questions but otherwise LGTM!

**Review by meganrm — APPROVED** [review]

File: src/simularium/RemoteSimulator.ts (hunk 2 above)
Comment (ascibisz) [reply src/simularium/RemoteSimulator.ts:21]: Correct! Octopus doesn't use this

**Pull request merged**